src/components/dashboard/DashBody.js

```diff
@@ -1,9 +1,11 @@
 import React, { Component } from 'react'
+
 import Text from './widgets/Text'
 import Image from './widgets/Image'
 import QRCode from './widgets/QRCode'
 import Clock from './widgets/Clock'
-import '../styles/dash_body.css'
+
+import '../../styles/dashboard/dash_body.css'
```

Comment:
Yes, we want to separate the order that things are imported to separate libraries, components and styles.
We also put css files into folders to match the components.